assets/js/admin/admin.js

```diff
@@ -1297,14 +1298,17 @@ jQuery(function ($) {
 		}
 	}, 2);
 
-	$(document).on('click', '#ur-tab-registered-fields h2', function () {
+	$(document).on('click', '.ur-toggle-heading', function () {
 		if ($(this).hasClass('closed')) {
```

Comment:
Always give add one blank line above if condition.

Reply:
@lekhnath005 Okay. I will do that right away and thank you for the review.